webapi/webapi-realsense-xwalk-tests/README.md

```diff
@@ -0,0 +1,25 @@
+# RealSense Test Suite
+
+## Introduction
+
+This test suite is for checking RealSense support in Crosswalk Project:
+* http://crosswalk-project.github.io/realsense-extensions-crosswalk/spec/depth-enabled-photography.html
+* http://crosswalk-project.github.io/realsense-extensions-crosswalk/spec/scene-perception.html
+* http://crosswalk-project.github.io/realsense-extensions-crosswalk/spec/face.html
+
+## Pre-Condition
+
+* Download and unzip realsense_extensions_v18.6.0.zip to realsense_extensions folder.
+https://github.com/crosswalk-project/realsense-extensions-crosswalk/releases/download/v18.6.0/realsense_extensions_v18.6.0.zip
+* Copy realsense_extensions folder to the root directory of realsense test suit.
```

Comment:
People still doesn't know how to pack the test suite since `pack.py` doesn't work.